v2/serverless_example.md

```diff
@@ -576,48 +574,55 @@ The next step is to create Lambda functions to create, show, and delete individu
 Replace the code in `widgets.js` \(in `resources`\) with the following\.
 
 ```js
-import {
+const {
   S3Client,
   ListObjectsV2Command,
   GetObjectCommand,
   PutObjectCommand,
-  DeleteObjectCommand
-} from '@aws-sdk/client-s3';
-
-// In the following code we are using AWS JS SDK v3
-// See https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html
+  DeleteObjectCommand,
+} = require("@aws-sdk/client-s3");
 const S3 = new S3Client({});
+
 const bucketName = process.env.BUCKET;
 
-exports.main = async function(event, context) {
+exports.main = async function (event, context) {
   try {
-    const method = event.httpMethod;
+    var method = event.httpMethod;
```

Comment:
Why var?

Reply:
That was changed by prettier. I don't mind adjusting it back to const, should be better

Reply:
Fixed, I accidentally changed it back from const to var from the original version